LICENSE

```diff
@@ -1,18 +1,4 @@
-Copyright 2024 Typelevel and respective contributors
-
-Licensed under the Apache License, Version 2.0 (the "License");
-you may not use this file except in compliance with the License.
-You may obtain a copy of the License at
-
-    http://www.apache.org/licenses/LICENSE-2.0
-
-Unless required by applicable law or agreed to in writing, software
-distributed under the License is distributed on an "AS IS" BASIS,
-WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
-See the License for the specific language governing permissions and
-limitations under the License.
-
---------------------------------------------------------------------------
+Copyright 2024 respective contributors
```

Comment:
I think Daniel meant, "keep `Copyright 2024 Typelevel` and drop the `respective contributors` part", but I don't think his firmware allows him to formulate that in a way that's not confusing to the rest of us, and my interpretation of his output stream may be wrong. 

So I'm referring myself to the source headers of [other typelevel projects](https://github.com/typelevel/cats-effect/blob/310150dc0b31ddcd6a528d46acf0d3e44cc02d8a/core/js/src/main/scala/cats/effect/IOApp.scala#L2C3-L2C33)